docs/deployment/Azure/configure.md

```diff
@@ -1,80 +1,172 @@
+  ecc_rule:
+    cordon_dry_run: False
+    reboot_dry_run: True
+    dri_email: DLTSDRI@microsoft.com
+    alert_job_owners: True
+    days_until_node_reboot: 5
+
+WebUIregisterGroups:
+- MicrosoftUsers
+
+WebUIauthorizedGroups : []
+WebUIadminGroups : ["CCSAdmins"]
+WebUIregisterGroups: [ "MicrosoftUsers" ]
+
+DeployAuthentications : ["Corp"]
+
+webuiport: 80
+
+cloud_config_nsg_rules:
+  default_admin_username: core
+  dev_network:
+    source_addresses_prefixes:
+    # These are the dev box of the cluster, only the machine in the IP address below will have access to the cluster.
+    - "b.a.0.0/16"
+    - "z.x.0.0/16"
+  nfs_share:
+    source_ips: 
+      # IPs that we want to share NFS storage to
+      - "x.y.z.0/24"
+      - "a.b.0.0/16"
+  nfs_ssh:
+    source_ips: 
+      # IPs that that we want to use to ssh to NFS nodes
+      - "q.w.e.0/24"
+      - "r.f.0.0/16"
+    port: "22"
+
+alert-manager:
+  configured: True
+  alert_users: False # True if we want to send out alert email to users, default False
+  smtp_url: <smtp url>
+  smtp_from: <email address used to send alert emails, e.g. 'dlts-bot@microsoft.com'>
+  smtp_auth_username: <email account that would send email to receivers, such as 'dlts-bot@microsoft.com'>
+  smtp_auth_password: <password for the email account above>
+  receiver: <email address to send alert email to>
+
+  reaper:
+    dry-run: True # change to False if we want to kill idle job
+    restful-url: http://localhost:5000
+
+prometheus:
+  cluster_name: <the unique cluster name> # will be used in link to job detail page
+
+registry_credential:
+  <docker registry name 1>:
+    username: <docker registry username 1>
+    password: <docker registry password 1>
+  <docker registry name 2>:
+    username: <docker registry username 2>
+    password: <docker registry password 2>
+```
+
+* cluster_name: A name without underscore or numbers (purely consisting of lower case letters) is recommended.
+
+* infra_node_num: Should be odd (1, 3 or 5), number of infrastructure node for the deployment. 3 infrastructure nodes tolerate 1 failure, and 5 infrastructure nodes tolerate 2 failures. However, more infrastructure nodes (and more failure tolerance) will reduce performance of the node. 
+
+* worker_node_num: Number of worker node used for deployment. 
+
+* vm_image: Used to fix the image version if the changing LTS is breaking the consistency of the deployment.
+
+* nfs_vm: each item identified by `suffix` specs would describe an NFS node, and this item would overwrite default NFS specs. A `server_suffix` entry in `nfs_mnt_setup` should map to this item.
+
+* azure_location: 
+
+Please use the following to find all available azure locations. 
+```
+az account list-locations
+```
+
+* infra_vm_size, worker_vm_size: infrastructure and worker VM size. 
+
+Usually, a CPU VM will be used for infra_vm_size, and a GPU VM will be used for worker_vm_size. Please use the following to find all available Azure VM size. 
```

Comment:
"... find all available Azure VM size in a specific region, e.g. West US 2 in the below command"